klein/test/test_resource.py

```diff
@@ -101,7 +104,7 @@ def getWrittenData():
 def _render(resource, request, notifyFinish=True):
     result = resource.render(request)
 
-    if isinstance(result, str):
+    if isinstance(result, bytes):
```

Comment:
I think we probably also need to be able to handle text being returned by rendering as well?